executor/runtime/docker/docker_ssh.go

```diff
@@ -135,21 +123,12 @@ func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cf
 		return err
 	}
 
-	// The format that is used for SSH Users is:
-	// $(unix username):$(app name):$(aws account id):$(task id)
-
+	containerEnv := c.Env()["NETFLIX_ENVIRONMENT"]
 	users := append(cfg.ContainerSSHDUsers, c.AppName())
 	for _, username := range users {
-		lines := []string{}
-		for _, accountID := range accountIDs {
-			lines = append(
-				lines,
-				fmt.Sprintf("%s:%s:%s:%s", username, c.AppName(), accountID, c.TaskID()), // key scoped to username, appname, account ID, and task ID
-				fmt.Sprintf("%s:%s:%s", c.AppName(), accountID, c.TaskID()),              // key has access to any username for this given app in this given account, with this task ID
-				fmt.Sprintf("%s:%s", c.AppName(), accountID),                             // key has access to any username for this given app in this given account
-				c.TaskID(), // key has access to any username on this task ID
-				fmt.Sprintf("%s:%s", username, c.TaskID()), // key has access to this given username on this task ID
-			)
+		lines := []string{
```

Comment:
We are already on v3 everywhere server-side?

Reply:
The 2.0 flow (go/ssh) is producing certs including the v3 principals now. Probably the bastion 1.0 / titus-ssh infrastructure is not, although that should be getting torn down entirely in the next few weeks.

Reply:
I believe some of the above principals were used by the titus team themselves for power user access. @solarkennedy do you know if y'all still use the app:account, taskId, or user:taskId formats for your custom CA tooling?

Removing support for the v1 line, app:account:taskID will prevent SSH Certs for tasks with non-spinnaker compatible stack/detail names from accepting certs issued to them.

Reply:
> Yeah, merging this would break the bastion 1.0 users, but if we need to support that before the tear down, let me know and we can patch it.

Yea, bastion 1.0 is to be torn down very imminently, so we can probably save the effort involved.

> Removing support for the v1 line, app:account:taskID will prevent SSH Certs for tasks with non-spinnaker compatible stack/detail names from accepting certs issued to them.

At least in the 2.0 flow we lookup the stack/detail via the titus API and put them in the cert, so even if it's a non-spinnaker compatible name as long as it doesn't have a colon it should still work.

> I believe some of the above principals were used by the titus team themselves for power user access.

Yea, that's the main think that's a bit unclear to me. If there's some other tooling making use of these principals I don't want to break that. But I pulled the thread on the trusted CA keys and it's just the bless keys, so I believe whatever tooling it is would have to be calling the 1.0 BLESS service.

Reply:
Ah, thanks for pulling the CA keys trust set. There was a time when "alternative" CAs were being added to the mix. Nothing on the bless CAs has used those other formats.